### PR TITLE
retroactive styling for #288

### PR DESCRIPTION
**Pull request by justinlaughlin**
#288 wasn't styled, this PR fixes that.

### CODE REVIEW

File: lib/threads.cpp [hunk 1]
```diff
@@ -1277,7 +1278,8 @@ void communication_thread::execute()
          string formatting;
 
          *is[0] >> ws >> c; // read the opening char
-         getline(*is[0], formatting, c); // read formatting string & use c for termination
+         getline(*is[0], formatting,
+                 c); // read formatting string & use c for termination
```
Comment (najlkin) [inline lib/threads.cpp:1282]: This is awful, maybe place the comment before the line (and above too) 😉 .

File: lib/vsdata.cpp [hunk 2]
```diff
@@ -571,10 +571,11 @@ void KeycPressed(GLenum state)
...
                                   &default_precision,
-                                  [](int p){ return p>=0; });
-      char format = prompt<char>("Enter format [(d)efault, (f)ixed, (s)cientific] (d): ",
-                                 &default_format,
-                                 [](char c){ return c=='d' || c=='f' || c=='s'; });
+      [](int p) { return p>=0; });
+      char format =
+         prompt<char>("Enter format [(d)efault, (f)ixed, (s)cientific] (d): ",
+                      &default_format,
+      [](char c) { return c=='d' || c=='f' || c=='s'; });
```
Comment (najlkin) [inline lib/vsdata.cpp:578]: Hmm, this looks like an error of astyle 🤔 Maybe try to put it at the end of the previous line?

**Comment by tzanio** [comment]
See https://github.com/GLVis/glvis/commit/c697e0b465988d5a0c082edf0813705ffac75131 in `glvis-4.3-dev`. Maybe we can make the changes directly there?

**Comment by najlkin** [comment]
Rather here, to get it to master quickly and stop autotest errors.

File: lib/vsdata.cpp [hunk 3]
```diff
@@ -628,10 +627,11 @@ void Key_Mod_a_Pressed(GLenum state)
...
```
Comment (najlkin) [inline lib/vsdata.cpp:634]: Here too 😇

**Comment by justinlaughlin** [comment]
> See [c697e0b](https://github.com/GLVis/glvis/commit/c697e0b465988d5a0c082edf0813705ffac75131) in `glvis-4.3-dev`. Maybe we can make the changes directly there?

I'm okay with either.

File: lib/vsdata.cpp (hunk 3 above)
Comment (najlkin) [reply lib/vsdata.cpp:634]: ☝️

Comment (justinlaughlin) [reply lib/vsdata.cpp:634]: fixed in https://github.com/GLVis/glvis/pull/299/commits/ed619725e66331dabd685d65e403b12b78747ea7

Comment (najlkin) [reply lib/vsdata.cpp:634]: nope

**Review by najlkin — APPROVED** [review]

File: lib/vsdata.cpp (hunk 3 above)
Comment (justinlaughlin) [reply lib/vsdata.cpp]: I can run the styler, or keep it how it was, but if you want something else I don't know what it is.

Comment (najlkin) [reply lib/vsdata.cpp]: You must fight it until it does not change the code, otherwise it will not pass the checks on master.

**Review by tzanio — APPROVED** [review]

**Pull request merged**